src/gui_common/TweakedColourPicker.tscn

```diff
@@ -1,443 +1,114 @@
-VScrollBar/styles/grabber = null
-VScrollBar/styles/grabber_highlight = null
-VScrollBar/styles/grabber_pressed = null
-VScrollBar/styles/scroll = null
-VScrollBar/styles/scroll_focus = null
-VSeparator/constants/separation = 4
-VSeparator/styles/separator = null
-VSlider/icons/grabber = null
-VSlider/icons/grabber_disabled = null
-VSlider/icons/grabber_highlight = null
-VSlider/icons/tick = null
-VSlider/styles/grabber_area = null
-VSlider/styles/grabber_area_highlight = null
-VSlider/styles/slider = null
-VSplitContainer/constants/autohide = 1
-VSplitContainer/constants/separation = 12
-VSplitContainer/icons/grabber = null
-VSplitContainer/styles/bg = null
-WindowDialog/colors/title_color = Color( 0, 0, 0, 1 )
-WindowDialog/constants/close_h_ofs = 18
-WindowDialog/constants/close_v_ofs = 18
-WindowDialog/constants/scaleborder_size = 4
-WindowDialog/constants/title_height = 20
-WindowDialog/fonts/title_font = null
-WindowDialog/icons/close = null
-WindowDialog/icons/close_highlight = null
-WindowDialog/styles/panel = null
+[ext_resource path="res://src/gui_common/thrive_theme.tres" type="Theme" id=1]
+[ext_resource path="res://src/gui_common/TweakedColourPicker.cs" type="Script" id=2]
+[ext_resource path="res://assets/textures/gui/bevel/plusButton.png" type="Texture" id=3]
+[ext_resource path="res://assets/textures/gui/bevel/plusButtonHover.png" type="Texture" id=4]
+[ext_resource path="res://assets/textures/gui/bevel/plusButtonDisabled.png" type="Texture" id=5]
 
 [node name="TweakedColourPicker" type="ColorPicker"]
-margin_left = 4.0
-margin_top = 4.0
-margin_right = 294.0
-margin_bottom = 492.0
-theme = SubResource( 1 )
+margin_left = 8.0
+margin_top = 8.0
+margin_right = 8.0
+margin_bottom = 8.0
+theme = ExtResource( 1 )
+script = ExtResource( 2 )
 __meta__ = {
 "_edit_use_anchors_": false,
-"_editor_description_": "A color picker with a custom theme."
+"_editor_description_": "A color picker with thrive theme and extended usage."
+}
+
+[node name="MarginButtonsContainer" type="MarginContainer" parent="."]
+margin_top = 494.0
+margin_right = 299.0
+margin_bottom = 541.0
+custom_constants/margin_top = 5
+
+[node name="ButtonsContainer" type="HBoxContainer" parent="MarginButtonsContainer"]
+margin_top = 5.0
+margin_right = 299.0
+margin_bottom = 47.0
+size_flags_horizontal = 3
+
+[node name="HSVCheckButton" type="CheckButton" parent="MarginButtonsContainer/ButtonsContainer"]
+margin_right = 101.0
+margin_bottom = 42.0
+theme = ExtResource( 1 )
+shortcut_in_tooltip = false
+action_mode = 0
+keep_pressed_outside = true
+text = "HSV"
+__meta__ = {
+"_editor_description_": "PLACEHOLDER"
+}
+
+[node name="RawCheckButton" type="CheckButton" parent="MarginButtonsContainer/ButtonsContainer"]
+margin_left = 105.0
+margin_right = 207.0
+margin_bottom = 42.0
+theme = ExtResource( 1 )
+shortcut_in_tooltip = false
+action_mode = 0
+keep_pressed_outside = true
+text = "Raw"
+__meta__ = {
+"_editor_description_": "PLACEHOLDER"
```

Comment:
Are these the actual buttons? Shouldn't these be translatable. I would bet that there is at least one language where it makes sense to translate these.

Reply:
Also using "raw" to denote a special kind of input, colour value, photo mode, is actually very common, and languages might have a convention on those.